projects/mercury/src/metadata/common/values/DateValue.js

```diff
@@ -34,7 +34,7 @@ class DateValue extends React.Component {
         return (
             <LocalizationProvider dateAdapter={AdapterDateFns}>
                 <DatePicker
-                    format={DATE_FORMAT}
```

Comment:
Why is `format` a wrong property name? The documentation specifies it like this and I don't see `inputFormat` there.
https://mui.com/x/api/date-pickers/date-picker/#DatePicker-prop-format

Reply:
it's been changed in the 6th version, but we use the 5th one, here is the proper link https://v5.mui.com/x/api/date-pickers/date-picker/

Reply:
ahh good catch!